lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/CardList.js

```diff
@@ -54,7 +83,31 @@ qx.Class.define("callbackery.ui.plugin.CardList", {
             this._loading++;
             rpc.callAsync(function(data,exc){
                 if (!exc){
-                    that.setData(data,true);
+                    if (Array.isArray(data)){
+                        // the data is an array of card data
+                        that.setData(data,true);
+                        that.setInfo(null);
+                    }
+                    else {
+                        // the data is an object with card data and info
+                        that.setData(data.data,true);
+                        if (data.info) {
+                            // label is the translation key
+                            // args is an array of arguments for the translation
+                            let args = data.info.args;
+                            delete data.info.args;
+                            if (args && args.length > 0) {
+                                data.info.label = that.xtr(data.info.label, ...args);
```

Comment:
hmmm ich bin grad etwas verwirrt warum dass du das explizit behandeln musst im frontend ... ich haette gedacht dass trm(string,a,b,c) reicht ...

Reply:
Du hast recht, `that.xtr(data.info.label)` reicht. Fixed.